3.2/Dockerfile.rhel7

```diff
@@ -63,13 +66,16 @@ ENV BASH_ENV=${CONTAINER_SCRIPTS_PATH}/scl_enable \
     ENV=${CONTAINER_SCRIPTS_PATH}/scl_enable \
     PROMPT_COMMAND=". ${CONTAINER_SCRIPTS_PATH}/scl_enable"
 
+COPY s2i/bin $STI_SCRIPTS_PATH
 COPY root /
 
 # this is needed due to issues with squash
 # when this directory gets rm'd by the container-setup
 # script.
 RUN /usr/libexec/container-setup
 
+RUN usermod -a -G root redis
```

Comment:
Is it needed to have this command in separate RUN statement? (it could be joined with statement above)